src/components/TransactionsInterpreter/helpers/capitalizeFirstLetter.ts

```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (str: string) => {
+  return str.length > 1 ? str.charAt(0).toUpperCase() + str.slice(1) : str;
```

Comment:
https://lodash.com/docs/4.17.15#capitalize

No need for one more util function